synapse/models/science.py

```diff
@@ -0,0 +1,92 @@
+import synapse.lib.module as s_module
+
+class ScienceModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('sci', {
+            'types': (
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', ('guid', {}), {
+                    'doc': 'A hypothesis or theory.'}),
+
+                # TODO link experiment to eventual procedure node
+                ('sci:experiment:type:taxonomy', {}, {}),
+                ('sci:experiment', ('guid', {}), {
+                    'doc': 'An instance of running an experiment to test a hypothesis.'}),
+
+                ('sci:observation', ('guid', {}), {
+                    'doc': 'An observation which may have resulted from an experiment.'}),
+
+                ('sci:evidence', ('guid', {}), {
+                    'doc': 'An assessment of how an observation supports or refutes a hypothesis.'}),
```

Comment:
Would evidence be "facts" and not an "assessment"?

Reply:
No, observations are facts. Evidence is the interpretation of those observations to support or refute a hypothesis.